source/ts/utils/httpUtils.ts

```diff
@@ -0,0 +1,76 @@
+/// <reference path="../vender/nanoajax.d.ts" />
+
+module Growthbeat {
+  export class HttpUtils {
+    urlUtils = new Growthbeat.UrlUtils();
+
+    get(url:string, isCredential?:boolean,
+      success?:(res:any)=>void, error?:(errorModel:Growthbeat.ErrorModel)=>void) {
+      Growthbeat.nanoajax.ajax({
+        url: 'http://api.growthbeat.local:8085' + url,
```

Comment:
The correct endpoint is "https://api.growthbeat.com/."
Add setEndpoint method to config test or production environment.